lib/media/track/remotevideotrack.js

```diff
@@ -47,23 +47,37 @@ class RemoteVideoTrack extends RemoteMediaVideoTrack {
       _documentVisibilityTurnOffCleanup: {
         value: null,
         writable: true
-      }
+      },
+      _intersectionObserver: {
+        value: new IntersectionObserver(entries => {
```

Comment:
we should add IntersectionObserver to options (at line 38 above) like: 
```
    options = Object.assign({
      enableDocumentVisibilityTurnOff: true,
     IntersectionObserver
    }, options);
```
and then this line becomes
```suggestion
        value: new options.IntersectionObserver(entries => {
```

This will allow us to mock it from unit tests